pkgs/development/compilers/ghc/8.8.2.nix

```diff
@@ -86,12 +86,12 @@ let
 
 in
 stdenv.mkDerivation (rec {
-  version = "8.8.1.20191211";
+  version = "8.8.2";
   name = "${targetPrefix}ghc-${version}";
 
   src = fetchurl {
-    url = "https://downloads.haskell.org/ghc/8.8.2-rc1/ghc-${version}-src.tar.xz";
-    sha256 = "1gl4fzakjbhd94v1saxmr9sfzgk22m1b95jq51rxm93b2g4cixl4";
+    url = "https://downloads.haskell.org/ghc/8.8.2/ghc-${version}-src.tar.xz";
+    sha256 = "1qf4nrrxn0fnfvxri5m34nx6f3x2px85xhxnypapa4j3y3dh5xvj";
```

Comment:
I got a different hash
```
$ nix-prefetch-url https://downloads.haskell.org/ghc/8.8.2/ghc-8.8.2-src.tar.xz
[18.5 MiB DL]
path is '/nix/store/7nzka6x2b5rvkjr27gdff2rhdcabdxnz-ghc-8.8.2-src.tar.xz'
02qa6wgjpxgakg7hv4zfdlrx9k7zxa5i02wnr6y9fsv8j16sbkh1
```

And the resulting ghci starts OK.  (I did no other testing.)

Reply:
Ugh, that's strange, let me double-check.

Reply:
I get this:
```
[nix-shell:~/nixpkgs]$ nix-prefetch-url --unpack https://downloads.haskell.org/ghc/8.8.2/ghc-8.8.2-src.tar.xz
unpacking...
[18.5 MiB DL]Warning: Failed to set locale category LC_NUMERIC to en_DE.
Warning: Failed to set locale category LC_TIME to en_DE.
Warning: Failed to set locale category LC_COLLATE to en_DE.
Warning: Failed to set locale category LC_MONETARY to en_DE.
Warning: Failed to set locale category LC_MESSAGES to en_DE.
[18.5 MiB DL]
path is '/nix/store/p4d9ly54n6iwaa27ksw1fl2g19l1lb0p-ghc-8.8.2-src.tar.xz'
1qf4nrrxn0fnfvxri5m34nx6f3x2px85xhxnypapa4j3y3dh5xvj
```

Reply:
I shouldn't pass `--unpack` then?

Reply:
No, not if it's for plain `fetchurl`.  That's what we use typically.

Reply:
I was blindsided by https://github.com/NixOS/nix/issues/1880#issuecomment-366615204, but that is for GitHub downloads. Gotcha!

Reply:
Version-control archives are typically generated dynamically and their (packed) hash changes over time, so plain `fetchurl` isn't good there.

Reply:
Makes sense.